tests/app/main/views/test_users.py

```diff
@@ -176,6 +186,27 @@ def test_should_show_deactivate_button(self):
         assert deactivate_button == 'Deactivate'
         assert return_link.attrib['value'] == '/admin/users?email_address=test.user%40sme.com'
 
+    @pytest.mark.parametrize('case', ({'active': False}, {'personalDataRemoved': True}))
```

Comment:
Go on, add the case where they're both deactivated *and* personal data's removed... very cheap extra test.